src/modules/discovery/pages/activity/components/ActivityFeedItem.tsx

```diff
@@ -357,7 +362,12 @@ const RewardsInfo = ({ reward }: { reward: ProjectReward }) => {
       <Body size="sm" bold>
         {formatAmount(reward.cost, reward.rewardCurrency)}{' '}
         <Body as="span" size="sm" muted>
-          {reward.rewardCurrency === RewardCurrency.Btcsat ? 'Sats' : 'USD'}
+          {reward.rewardCurrency === RewardCurrency.Btcsat ? 'Sats' : 'USD'}{' '}
```

Comment:
let's use "sats" small case, I think vlad added "Sats" in the design but we all decided small sats is more used in the community, and we should follow that.